homework-1/index.html

```diff
@@ -0,0 +1,681 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Scout</title>
+  <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css" />
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <div id="banner-submenu" class="fixed-top">
+    <nav class="submenu">
+      <ul class="submenu__list list-top" id="list-top">
+        <li class="first-item"><a class="tips-active" href="#tips">Tips</a></li>
+        <li class="submenu__item-middle"><a class="submenu__link" href="#glossary">Glossary</a></li>
+        <li class="last-item"><a href="#fqas">FAQs</a></li>
+      </ul>
+    </nav>
+  </div>
+  <header class="header" id="header">
```

Comment:
there is no reason to use id - header here

Reply:
fixed a bug